content/software-security/glossary.md

```diff
@@ -57,13 +63,19 @@ Provenance is the verifiable information about software artifacts describing _wh
 
 ### Certificate authority
 
-Often abbreviated as **CA**, a certificate or certification authority is a governing body that stores, signs, and issues digital certificates that can verify claims about the ownership of a given public key. Software consumers can use a CA to verify the assertions made about the private key that corresponds with the certified public key. A trusted third party, CAs use the X.509 or EMV standard to format certificates. 
+Often abbreviated as **CA**, a certificate or certification authority is a governing body that stores, signs, and issues digital certificates that can verify claims about the ownership of a given public key. Software consumers can use a CA to verify the assertions made about the private key that corresponds with the certified public key. A trusted third party, CAs use the X.509 or EMV standard to format certificates.
+
+---
+
+### CI/CD
```

Comment:
I would probably put this in the top section, CI/CD is not necessarily security focused. I think we are operating under the assumption that most developers know this one, but not against adding it

Reply:
Honestly, I did not know what it meant a few weeks ago, so it could be just me! I guess it doesn't hurt to add, though. I can move it up, I agree it fits there better.